README.md

```diff
@@ -16,13 +16,6 @@ Additional description about the project and its features.
 - Frameworks: NO
 - Technologies used: Github and git flow, VS code
 
-<<<<<<< HEAD
-=======
-## Live Demo
-
-[Live Demo Link](https://oyelaking9.github.io/leaderBoard/dist/)
-
->>>>>>> d901839a65cf5e32099b0bf0380af889a095b3da
 ## Getting Started:
  Run: npm install (To install all npm package)
  Run: npm run started (To start the code in browser).
```

Comment:
- [ ] This line should be `npm start` instead of `npm started`
- [ ]  _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** 
- [ ] The _code_ aspects in each procedure should be quoted with **backticks**.